packages/scene-composer/src/components/panels/SceneHierarchyPanel/components/SceneHierarchyTree/SceneHierarchyTreeItem.tsx

```diff
@@ -47,7 +48,8 @@ const SceneHierarchyTreeItem: FC<SceneHierarchyTreeItemProps> = ({
       (type as unknown as IModelRefComponentInternal)?.modelType !== ModelType.Environment,
   );
 
-  const showSubModel = isValidModelRef && !!model && !isViewing();
+  const [{ variation: subModelSelectionEnabled }] = useFeature(COMPOSER_FEATURES[COMPOSER_FEATURES.SubModelSelection]);
```

Comment:
Can you just pass `useFeature(COMPOSER_FEATURES.SubModelSelection)`? Curious if the extra nesting is really necessary.

Reply:
I'm following how useFeature is used for other features, not quite sure why we did this.... probably because this COMPOSER_FEATURES enum used to be int enum not string enum..